### PR TITLE
Checkpoint 5 items

### PR DESCRIPTION
**Pull request by TWTY1STC**

### CODE REVIEW

**Comment by TWTY1STC** [comment]
I'm prepared to merge all this onto master, but I could use a second look over. Thank you for your notes and comments on Items_model branch. I rebased that branch before working on it again, but when I went to push it back to GitHub I got a strange error that I wasn't sure how to resolve, so I created this new branch. 
Prior to seeing your notes on Items_model, I created a branch to set up gems (factorygirl, rspec, faker). You'll notice this branch includes some of that set up work because of the rebasing. 

The form was rendering in #show view, but the items partial was tricky. First, I realized that nothing was being saved onto the database (my create method needed many tweaks), and, second, the code to display it was broken. It was pretty exciting after many attempts to see the new lists save correctly and render on the show page. 

Essentially, this will complete checkpoint 5, and, perhaps, 6 because I seeded the database. I need to double check that the associations make sense. I also want to spend some time making the app visually appealing, for example, now that the items are rendering, I'd like to have them render onto a bootstrap table.

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  
+  def create
+    @item = current_user.items.new(item_params)
+    
+    if @item.save
+      flash[:notice] = "Item has been successfully added to your list"
+      redirect_to root_path
+    else
+      flash[:alert] = "There was an error saving your list item. Please try again"
+      render 'items/form'
```
Comment (cbeck) [inline app/controllers/items_controller.rb]: I don't think you want to do this. Just redirect to root path here too.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -3,8 +3,15 @@
...
 <p><strong>Email:</strong> <%= current_user.email %></p>
 
-<%= link_to "Edit Profile" , edit_user_registration_path, :class => 'btn btn-success'%>
+<%= link_to "Edit Profile", edit_user_registration_path, :class => 'btn btn-success'%>
 
-<h1> On your list</h1>
+<h1><strong>Your list</strong> </h1>
 
+<div class="media"
+  <%= render "items/item" %>
```
Comment (cbeck) [inline app/views/users/show.html.erb]: Should be

```
<%= render current_user.items %>
```

File: app/views/items/_item.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+<% current_user.items.each do |item| %>
```
Comment (cbeck) [inline app/views/items/_item.html.erb]: Take the loop out of this - we only put the code to display one item in the partial.

File: app/views/items/_item.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  <h3><%= item.name %></h3>
+   
+    <small>
+        created <%= time_ago_in_words(item.created_at) %> ago 
+    </small>
+<% end %>
```
Comment (cbeck) [inline app/views/items/_item.html.erb]: Remove

File: spec/rails_helper.rb [hunk 5]
```diff
@@ -0,0 +1,63 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+# Add additional requires below this line. Rails is not loaded until this point!
+
+<<<<<<< 868844396fb0f704a5794f17fd7538db6d85178a
```
Comment (cbeck) [inline spec/rails_helper.rb]: Delete this line

File: spec/rails_helper.rb [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+require 'factory_girl_rails'
+
+=======
```
Comment (cbeck) [inline spec/rails_helper.rb]: Delete lines 13 and 14

**Pull request merged**